### PR TITLE
Mapping OhioStation

### PR DESCRIPTION
**Pull request by JasperStingtail**
# Description

Adds in a new map: OhioStation
Also adds one new statue for the map.\

:cl:
- add: Added Ohio

### CODE REVIEW

File: Resources/Prototypes/Maps/ohiostation.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+﻿- type: gameMap
+  id: OhioStation
+  mapName: 'OhioStation'
+  mapPath: /Maps/ohiostation.yml
+  minPlayers: 0
+  stations:
+    OhioStation:
+      mapNameTemplate: '{0} Ochd Deug oh Tri {1}'
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Maps/ohiostation.yml:8]: what

**Comment by DEATHB4DEFEAT** [comment]
put a maprender in the media if the renderer works
why Ohiostation :agony:

File: Resources/Prototypes/Maps/ohiostation.yml (hunk 1 above)
Comment (JasperStingtail) [reply Resources/Prototypes/Maps/ohiostation.yml:8]: Scottish

Comment (DEATHB4DEFEAT) [reply Resources/Prototypes/Maps/ohiostation.yml:8]: what is it

Comment (JasperStingtail) [reply Resources/Prototypes/Maps/ohiostation.yml:8]: Its eighteen oh three

Comment (DEATHB4DEFEAT) [reply Resources/Prototypes/Maps/ohiostation.yml:8]: why

**Comment by JasperStingtail** [comment]
ohiostation-0

The render

**Comment by DEATHB4DEFEAT** [comment]
arborum?

**Comment by JasperStingtail** [comment]
mistake

**Comment by JasperStingtail** [comment]
Maprender doesn't work. Only outputs ParkStation.

File: Resources/Prototypes/Maps/ohiostation.yml (hunk 1 above)
Comment (JasperStingtail) [reply Resources/Prototypes/Maps/ohiostation.yml:8]: Its funny

**Review by Pspritechologist — APPROVED** [review]

**Pull request merged**